src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,21 @@
-export const UserInfo = () => {};
+import React from 'react';
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
+
+  return (
+    <a className="UserInfo" href={`mailto:${email}`}>
```

Comment:
Consider using `Link` component from `react-router-dom` instead of `a` tag for better accessibility and user experience. 🔗